mapstruct-on-gradle-testcp-with-lombok/build.gradle

```diff
@@ -0,0 +1,25 @@
+plugins {
+    id 'java'
+    id "io.freefair.lombok" version "6.0.0-m2"
```

Comment:
Why use a plugin and not add lombok to the dependencies in a normal way?

Reply:
The plugin handles the annotation processor and dependencies. It’s actually recommended at the lombok site.

Reply:
[https://projectlombok.org/setup/gradle](https://projectlombok.org/setup/gradle)

Reply:
Just saw this as well. Didn't know about this plugin. Since it ia suggested by Lombok itself let's keep it and also update the other examples with it

Reply:
I have not removed the direct configuration, but added a hint, that the other example uses the plugin. As both ways are official, and someone might have compliance restrictions on using another plugin, both might be helpful and should be preserved.

Reply:
It makes sense. Thanks for doing the updates